jsonnet/telemeter-client.libsonnet

```diff
@@ -28,6 +28,13 @@ function(params) {
   secret: tc.telemeterClient.secret,
   servingCertsCABundle: tc.telemeterClient.servingCertsCABundle,
   deployment: tc.telemeterClient.deployment {
+    metadata+: {
+      labels+: {
+        'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        'app.kubernetes.io/component': 'telemetry-metrics-collector',
+        'app.kubernetes.io/name': 'telemeter-client',
```

Comment:
These two should be moved to https://github.com/openshift/telemeter/pull/385/ and we should add  `'app.kubernetes.io/part-of': 'openshift-monitoring'` here. The goal is to have everything generic "upstream" and everything CMO specific here.